### PR TITLE
assign unique names to symbolic inputs

### PR DESCRIPTION
**Pull request by jiaqi-liu**

### CODE REVIEW

File: lib/Transforms/Instrumentation/KleeInternalize.cc [hunk 1]
```diff
@@ -239,13 +243,13 @@ namespace
 
     bool runOnModule (Module &M)
     {
+      m_module = &M;
       declareKleeFunctions(M);
       internalizeVariables(M);
 
       for (Function &F : M)
       {
-        if (shouldInternalize (F)) defineFunction (F);
```
Comment (agurfinkel) [inline lib/Transforms/Instrumentation/KleeInternalize.cc]: why did you remove this line?

Comment (agurfinkel) [inline lib/Transforms/Instrumentation/KleeInternalize.cc]: Is that because now the function is defined at a call site, so that each call site has a different definition?

File: lib/Transforms/Instrumentation/KleeInternalize.cc [hunk 2]
```diff
@@ -270,7 +274,22 @@ namespace
           CallInst *ninst = nullptr;
           Builder.SetInsertPoint (&inst);
           
-          if (fn->getName().equals ("verifier.assume"))
+          if (shouldInternalize(*fn))
+          {
+            int id = m_functionId[fn->getName()];
+            Function * newFn = cast<Function> (m_module->getOrInsertFunction(
+              fn->getName().str() + "_" +std::to_string(id),
```
Comment (agurfinkel) [inline lib/Transforms/Instrumentation/KleeInternalize.cc]: use a period instead of underscore to prevent clashes with user defined functions.

Comment (jiaqi-liu) [inline lib/Transforms/Instrumentation/KleeInternalize.cc]: Good point. Changed.

File: lib/Transforms/Instrumentation/KleeInternalize.cc (hunk 1 above)
Comment (jiaqi-liu) [inline lib/Transforms/Instrumentation/KleeInternalize.cc]: Yes. We need to assign a unique name per call. I added a few lines of code to remove the initial declaration in the new commit.
